Resources/Prototypes/_White/Catalog/uplink_catalog.yml

```
@@ -50,4 +50,15 @@
+  saleLimit: 1
+
+- type: listing
+  id: UplinkSmokeImplanter
+  name: uplink-smoke-implant-name
+  description: uplink-smoke-implant-desc
+  icon: { sprite: /Textures/Objects/Weapons/Grenades/smoke.rsi, state: icon }
+  productEntity: SmokeImplanter
+  cost:
+    Telecrystal: 2
+  categories:
+  - UplinkImplants
```

yamllint

[warning] wrong indentation: expected 4 but found 2

(indentation)